topdown/internal/jwx/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: realclean cover viewcover
```

Comment:
Do we need the separate makefile and .gitignore?

Reply:
Some targets generate coverage files. These are very handy for now.